pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "msglint-1.04";
```

If using both `pname` and `version`, the version number within the `pname` can be omitted.

```suggestion
  pname = "msglint";
```

---

I'm really stunned! :)